Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -1611,6 +1608,9 @@ private void ActAccessibility(ActAccessibilityTesting act)
                     axeResult = axeBuilder.Analyze();
                 }
 
+                string path = $"{WorkSpace.Instance.Solution.ContainingFolderFullPath}\\{act.ItemName} AxeReport.html";
```

Comment:
use Path.PathseparatorCharactor instead of \\ as it is cross platform supported

Reply:
updated